### PR TITLE
Add Squadcast Integration Tile

### PR DESCRIPTION
**Pull request by dsahni**
### What does this PR do?

Adds an integration tile for Squadcast. 

### Motivation

Squadcast (tech partner) have created an integration on their platform. This PR adds a tile to document the setup process.

### CODE REVIEW

**Comment by dsahni** [comment]
cc @AlexandreYang to review

File: squadcast/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Use the Datadog-Squadcast integration to send Datadog alerts to Squadcast and seamlessly take actions on these alerts within the Squadcast platform.
+
+Connect Squadcast to Datadog in order to:
+- Trigger and resolve incidents from Datadog
+- Tackle incidents and set up escalation policies as they occur
+- Set up a daily reminder of who is on-call
+
+## Setup
+
+**Please Note:**
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
**Note**:
```

File: squadcast/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Only the users with Account Owner or Admin privileges can configure Services on Squadcast.
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
Only the users with Account Owner or Admin privileges can configure services on Squadcast.
```

File: squadcast/README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+At least one Escalation Policy must be configured before you can add a service.
+
+### Follow these steps on Squadcast:
+
+1. Open Services page from the sidebar
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
1. Open Services page from the sidebar.
```

File: squadcast/README.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+2. Click on "Add Service"
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
2. Click on "Add Service".
```

File: squadcast/README.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+3. Enter a meaningful Service Name and optionally a Service Description
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
3. Enter a meaningful **Service Name** and optionally a **Service Description**.
```

File: squadcast/README.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+4. Select Datadog from the Integration Type drop-down menu
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
4. Select **Datadog** from the **Integration Type** drop-down menu.
```

File: squadcast/README.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+5. Copy the Datadog Webhook URL generated below and click "Save"
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
5. Copy the **Datadog Webhook URL** generated below and click "Save".
```

File: squadcast/README.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+- Tackle incidents and set up escalation policies as they occur
+- Set up a daily reminder of who is on-call
+
+## Setup
+
+**Please Note:**
+Only the users with Account Owner or Admin privileges can configure Services on Squadcast.
+At least one Escalation Policy must be configured before you can add a service.
+
+### Follow these steps on Squadcast:
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
### Follow these steps in Squadcast:
```

File: squadcast/README.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+3. Enter a meaningful Service Name and optionally a Service Description
+
+4. Select Datadog from the Integration Type drop-down menu
+
+5. Copy the Datadog Webhook URL generated below and click "Save"
+
+![Squadcast Service][1]
+
+### Follow these steps on Datadog:
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
### Follow these steps in Datadog:
```

File: squadcast/README.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+1. Open Integrations page from the sidebar
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
1. Open the **Integrations** page from the sidebar.
```

File: squadcast/README.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+
+2. Use the search bar to search for Webhooks
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
2. Use the search bar to search for "webhooks".
```

File: squadcast/README.md [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+
+3. Once the Webhooks tile appears, hover and click on "Install"
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
3. Once the **Webhooks** tile appears, hover and click on "Install".
```

File: squadcast/README.md [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+
+4. Navigate to the Configuration tab and scroll to the bottom of the page
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
4. Navigate to the **Configuration** tab and scroll to the bottom of the page.
```

File: squadcast/README.md [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+
+5. Under the section Name and URL, enter a meaningful name and paste the Datadog Webhook URL provided by Squadcast
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
5. Under the section **Name and URL**, enter a meaningful name and paste the **Datadog Webhook URL** provided by Squadcast.
```

File: squadcast/README.md [hunk 15]
```diff
@@ -0,0 +1,82 @@
...
+
+    ![Squadcast Webhook][2]
+
+6. Tick the checkbox under the section Use custom payload
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
6. Tick the checkbox under the section **Use custom payload**.
```

File: squadcast/README.md [hunk 16]
```diff
@@ -0,0 +1,82 @@
...
+Copy-paste the following JSON in the text box under the Custom Payload section:
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
7. Copy-paste the following JSON in the text box under the **Custom Payload** section:
```

File: squadcast/README.md [hunk 17]
```diff
@@ -0,0 +1,82 @@
...
+    {
+        "alertId": "$ALERT_ID",
+        "eventMessage": "$TEXT_ONLY_MSG",
+        "title": "$EVENT_TITLE",
+        "url": "$LINK",
+        "alertTransition": "$ALERT_TRANSITION"
+    }
+    ```
+
+7. Click on "Install Integration" to complete the service integration
```
Comment (cswatt) [inline squadcast/README.md]:
```suggestion
8. Click on "Install Integration" to complete the service integration.
```

File: squadcast/README.md [hunk 18]
```diff
@@ -0,0 +1,82 @@
...
+
+    That's it! You are now good to go with your Datadog integration.
```
Comment (cswatt) [inline squadcast/README.md]: remove this line, it's not in the same tone as the rest of our docs.

File: squadcast/README.md [hunk 19]
```diff
@@ -0,0 +1,82 @@
...
+        "eventMessage": "$TEXT_ONLY_MSG",
+        "title": "$EVENT_TITLE",
+        "url": "$LINK",
+        "alertTransition": "$ALERT_TRANSITION"
+    }
+    ```
+
+8. Click on "Install Integration" to complete the service integration.
+
+    That's it! You are now good to go with your Datadog integration.
```
Comment (AlexandreYang) [inline squadcast/README.md]:
```suggestion
```

**Pull request merged**